Code_Exercises/Matrix_Transpose/README.md

```diff
@@ -0,0 +1,47 @@
+## Exercise: Local Memory Matrix Transpose
+---
+
+This exercise uses GPU specific features in order to gain good GPU performance.
+While code will run on other hardware, the local memory tiled solution may not
+give as good performance as it does on a GPU.
+
+In this exercise you will learn how to cache global memory into local memory in
+tiles according to work-groups in order to compare the performance difference.
+
+In order to get good performance, make sure that global memory accesses are
+coalesced. Local memory accesses do not have as large a penalty for
+uncoalesced memeory accesses.
+
+---
+
+Compile with
+
+For DPC++ (using the Intel DevCloud):
+```sh
+dpcpp -I../../Utilities/include source.cpp
+
+```
+For DPC++ (not Intel DevCloud):
+```sh
+clang++ -fsycl -fsycl-targets=<triple> -I../../Utilities/include source.cpp
```

Comment:
Maybe this should be `icpx` (maybe both?)

Reply:
Yep good catch thanks